documents/invoice/validator.go

```diff
@@ -22,7 +23,7 @@ func fieldValidator() documents.Validator {
 
 		var err error
 		if !documents.IsCurrencyValid(inv.Currency) {
-			err = documents.AppendError(err, documents.NewError("inv_currency", "currency is invalid"))
+			err = errors.AppendError(err, documents.NewError("inv_currency", "currency is invalid"))
```

Comment:
Since we deprectaed document.Error, we dont need to have key map errror. Prefer it if you use errors.New

Reply:
It would be in a different pr, same reason as explained before